scripts/close_inactive.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+require 'octokit'
+require 'pp'
+require 'yaml'
+
+CONFIG = {}
+CONFIG.merge!(YAML.load_file('config/close_inactive.yaml')) if File.exists?('config/close_inactive.yaml')
```

Comment:
Just let this fail if the config's missing, it's going to break if those settings aren't defined?